src/base-apis.js

```diff
@@ -1888,6 +1902,25 @@ MatrixBaseApis.prototype.getThirdpartyUser = function(protocol, params) {
     );
 };
 
+MatrixBaseApis.prototype.getTerms = function(serviceType, baseUrl, accessToken) {
+    const url = termsUrlForService(serviceType, baseUrl);
+    return this._http.requestOtherUrl(
+        undefined, 'GET', url, null, null, null,
```

Comment:
Could we instead omit the last 3 `null` args or are they needed for some reason?